### DISCUSSION TITLE
Tailwind prose plugin with catppuccin colors

### DISCUSSION THREAD

**Discussion by useEffects** [catppuccin/catppuccin#2458] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://github.com/tailwindlabs/tailwindcss-typography?tab=readme-ov-file#element-modifiers

### Any additional comments?

THE PORT WOULD BE HIGHLY OPINIONATED

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by uncenter** [comment]
Does https://github.com/catppuccin/tailwindcss not work for you? This should be an issue on that repository if not.

**Reply by uncenter** [reply]
It actually already has an issue sorry, https://github.com/catppuccin/tailwindcss/issues/13.

**Reply by useEffects** [reply]
So its better to work on the already existing tailwind repo, ill try to contribute
or lets make a separate port?

**Reply by uncenter** [reply]
I don't see why this has to be separate. There was a pull request that Winston was working on before leaving that you can look at for a start.
